docs/source/index.rst

```diff
@@ -57,10 +54,18 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    torch.utils.dlpack <dlpack>
    torch.utils.model_zoo <model_zoo>
    torch.utils.tensorboard <tensorboard>
-   type_info
+   torch.__config__ <__config__>
+   rpc
    named_tensor
    name_inference
-   torch.__config__ <__config__>
+   quantization
+   tensor_attributes
+   tensor_view
+   type_info
```

Comment:
Since this PR is just breaking up nn.rst into separate files, it shouldn't have the doc reorganization changes